src/main/java/Chat/tasks/Evelyn.java

```diff
@@ -0,0 +1,261 @@
+        }
+        echo();
+    }
+    private static void delete(String line){
+        try {
+            int index = Integer.parseInt(line.substring(7).trim()) - 1;
+            try {
+                tasks.remove(index);
+                int add1toIndex = index + 1;
+                System.out.println("Ok! Task " + add1toIndex + " is removed from the list");
+            } catch (ArrayIndexOutOfBoundsException e){
+                System.out.println("Invalid task index, please try again!");
+            }
+        } catch (NumberFormatException e){
+            System.out.println("Please key in a number after 'mark'");
+        }
+    }
+    private static void event(String line) {
+        boolean haveFrom = line.contains("/from");
+        boolean haveTo = line.contains("/to");
+        if (haveFrom && haveTo) {
+            try {
+                String[] parts = line.substring(6).split("/from");
+                String description = parts[0].trim();
+                String[] date = parts[1].trim().split("/to");
+                String from = date[0].trim();
+                String to = date[1].trim();
+                tasks.add(new Events(description, from, to));
+                indexOfTask++;
+                printAddingWords();
+                printLine();
+            } catch (ArrayIndexOutOfBoundsException e){
+                System.out.println("Please enter a description for the event!");
+            }
+        } else {
+            System.out.println("Please enter a time in the format of '/from (time) /to (time)'");
+        }
+    }
+
+    private static void deadline(String line) {
+        boolean haveBy = line.contains("/by");
+        if (haveBy) {
+            try {
+                String[] parts = line.substring(9).split("/by");
+                String description = parts[0].trim();
+                String by = parts[1].trim();
+                tasks.add(new Deadlines(description, by));
+                indexOfTask++;
+                printAddingWords();
+                printLine();
+            } catch (ArrayIndexOutOfBoundsException e){
+                System.out.println("Please enter a description for the deadline!");
+            }
+        } else {
+            System.out.println("Please enter a time in the format of 'by/ (time)'");
+        }
+    }
+
+    private static void todo(String line) {
+        try {
+            tasks.add(new Todos(line.substring(5).trim()));
+            indexOfTask++;
+            printAddingWords();
+            printLine();
+        } catch(StringIndexOutOfBoundsException e){
+            System.out.println("Please enter a description for the todo task!");
+        }
+    }
+
+    private static void unmark(String line) {
+        try {
+            int index = Integer.parseInt(line.substring(7).trim()) - 1;
+            try {
+                markTask(index, false);
+            }
+            catch (RepeatMark | RepeatUnmark e){
```

Comment:
Take note of the styling of where the catch should be. It should be after the } and not on a new line